pkg/polaris/creater.go

```diff
@@ -196,6 +199,22 @@ func GetPolarisBaseSecrets(polaris Polaris) (map[string]runtime.Object, error) {
 	return mapOfUniqueIDToBaseRuntimeObject, nil
 }
 
+func GetVersions(baseURL string) ([]string, error) {
+	serverURL, err := url.Parse(baseURL)
+	if err != nil {
+		return nil, err
+	}
+
+	serverURL.Path = path.Join(serverURL.Path, "polaris", "VERSIONS")
```

Comment:
do we need to import a package just to do "url/subpath/subpath"? 😄

Reply:
Let's not bikeshed about imports. The alternative is using fmt (which is not currently imported) and using `path` is a more elegant solution when dealing with paths.